hera_cal/frf.py

```diff
@@ -749,6 +1170,17 @@ def tophat_frfilter_argparser(mode='clean'):
     ap.add_argument("--max_frate_coeffs", type=float, default=None, nargs=2, help="Maximum fringe-rate coefficients for the model max_frate [mHz] = x1 * EW_bl_len [ m ] + x2."
                                                                                   "Providing these overrides the sky-based fringe-rate determination! Default is None.")
     ap.add_argument("--skip_autos", default=False, action="store_true", help="Exclude autos from filtering.")
+    ap.add_argument("--uvbeam", default=None, type=str, help="Path to UVBeam beamfits file to use for determining isotropic sky fringe-rates to filter.")
```

Comment:
If you want to avoid adding a new script too, perhaps have the associated script call the new function only when uvbeam is provided.